cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -649,6 +649,11 @@ fn aarch64_get_operands<F: Fn(VReg) -> VReg>(inst: &Inst, collector: &mut Operan
             collector.reg_def(rd);
             collector.reg_use(rm);
         }
+        &Inst::MovPReg { rd, .. } => {
+            collector.reg_def(rd);
+            // TODO FITZGEN: should we assert that `rm` is not an allocatable
```

Comment:
Ah! I see you had the same thought I did, reading a bit further :-)

I think the answer is definitely yes!